scripts/push-changelog-changes.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+EMAIL=$1
+UNAME=$2
+MSG=$3
+
+if [[ `git status --porcelain` ]]; then
+    git config --local user.email $1
+    git config --local user.name $2
+    git add CHANGELOG.md
```

Comment:
I kept this specific to changelog as if we do have automation pushing other files I want to be intentional about it.